### PR TITLE
Fix ol.css import

### PR DESCRIPTION
**Pull request by haslinghuis**

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

File: gulpfile.js
Comment (blckmn) [inline gulpfile.js:380]: why remove the indentation?

**Review by blckmn — CHANGES_REQUESTED** [review]

File: gulpfile.js
Comment (blckmn) [inline gulpfile.js:381]:
```suggestion
        .pipe(gulp.dest(`${DIST_DIR}node_modules/ol`));
```

File: gulpfile.js
Comment (blckmn) [inline gulpfile.js:380]:
```suggestion
        .src("./node_modules/ol/ol.css")
```

**Review by blckmn — APPROVED** [review]

**Comment by haslinghuis** [comment]
Applying https://github.com/betaflight/betaflight-configurator/pull/3196 would help

**Review by chmelevskij — CHANGES_REQUESTED** [review]
I’m pretty sure this will mess up web build.  Will fix tonight as I’ve had a stab at it recently

**Comment by haslinghuis** [comment]
Closing in favor of #3711

**Pull request closed without merging**
